features/0453-issue-credential-v2/README.md

```diff
@@ -10,6 +10,10 @@
 
 ## Version Change Log
 
+### 2.2 - Addition of Supplements
+
+An optional mechanism for providing credential supplements during issuance. Supplements are also used in credential presentation.
```

Comment:
@TelegramSam -- sorry for the delay in going through this, especially because I want to get this PR merged so that I can reference it in my own RFC :-).

I think this RFC needs more content here about what a supplement is for -- clarity. An example of how to use both of the types of supplements would be good -- especially a hashlink example, making clear how the key/value pair(s?) are used. An issuer-credential example would be good as well.